main.go

```diff
@@ -52,13 +53,24 @@ func main() {
 
 	fmt.Printf("Fetching last %d days of data (created>=%s)\n", since, created.Format("2006-01-02"))
 
-	var allRepos []*github.Repository
+	var repos, allRepos []*github.Repository
+	var res *github.Response
+	var err error
 	ctx := context.Background()
 	page := 0
 	for {
-		opts := &github.RepositoryListByOrgOptions{ListOptions: github.ListOptions{Page: page, PerPage: 100}, Type: "all"}
 		log.Printf("Fetching repos %s page %d", orgName, page)
-		repos, res, err := client.Repositories.ListByOrg(ctx, orgName, opts)
+		if orgName != "" {
+			opts := &github.RepositoryListByOrgOptions{ListOptions: github.ListOptions{Page: page, PerPage: 100}, Type: "all"}
+			log.Printf("Fetching repos %s page %d", orgName, page)
+			repos, res, err = client.Repositories.ListByOrg(ctx, orgName, opts)
+		}
+
+		if userName != "" {
```

Comment:
should this be an else if, to ensure that both requests can't ever be sent out? (same throughout)

Reply:
it's being handled right after the input https://github.com/self-actuated/actions-usage/pull/3/files#diff-2873f79a86c0d8b3335cd7731b0ecf7dd4301eb19a82ef7a1cba7589b5252261R38. 

I prefer the logic where you handle all the invalid inputs first and then you don't care about it later on during the execution.

Reply:
fair enough. i prefer not assuming that it's handled elsewhere, and validating inputs in every single function, in case they're reused in the future.